pkgs/development/python-modules/anthropic/default.nix

```diff
@@ -15,9 +15,9 @@
   pytest-asyncio,
   pytestCheckHook,
   pythonAtLeast,
-  pythonOlder,
   respx,
   sniffio,
+  stdenvNoCC,
```

Comment:
I know it doesn’t matter given we’re just evaluating the platform, but perhaps we should use `stdenv` here just as a matter of convention?

Reply:
It doesn’t pull `stdenv` into the closure though, it’s just for evaluation purposes.

Reply:
Thank you. In any case, it's moot now as I was able to remove that test.